lblod/frontend-subsidie-loket
# Task: remove unused code

## ID
DGS-138

## Description

Remove code not relevant to subsidy module. 

I have kept some of the lpdc and verenigingen code, since they might be needed in the future but not sure about it.

## Type of change


## How to test

Run the frontend with `ember serve --proxy https://loket.lblod.info/` and verify the subsidy module is still working.
